components/dashboard/src/Login.tsx

```diff
@@ -126,6 +136,12 @@ export function Login() {
                         </div>
                     </div>
                 </div>
+                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
+                    {`Email address is already in use. Please log in with ${emailTaken.otherUser.authHost}.`}
```

Comment:
**suggestion**: Ideally, we could have a title text for briefly explaining what happened and a body text to clarify what needs to be done. This way we can also make the title more prominent. We can tackle this separately and leave this as is for now in one piece.